packages/web-twig/src/Twig/SvgExtension.php

```diff
@@ -133,68 +131,99 @@ public function getInlineSvg(Environment $environment, string $path, array $para
      * @param array<string, string|array<string>> $params
      * @return false | SimpleXMLElement
      */
-    private function makeRegularSvg(?Source $source, ?string $iconId, array $params = [])
+    private function makeRegularSvg(?Source $source, ?string $iconId, array $params = [], ?string $symbolName = '')
     {
         if (! $source instanceof Source) {
             return false;
         }
 
         $svgString = $source->getCode();
+        $svg = @simplexml_load_string($svgString);
+
+        if ($svg === false) {
+            $this->logger->error('Error parsing SVG by simplexml_load_string from {class} in path "{path}"', [
+                'class' => Source::class,
+                'path' => $source->getPath(),
+            ]);
+            return false;
+        }
+
+        $this->applyAttributes($svg, $params, $iconId);
+
+        if ($symbolName && $symbolName !== '') {
+            $newSvg = new SimpleXMLElement('<svg xmlns="http://www.w3.org/2000/svg" style="display: none;"></svg>');
```

Comment:
```suggestion
            $svgWithSymbol = new SimpleXMLElement('<svg xmlns="http://www.w3.org/2000/svg" style="display: none;"></svg>');
```

or `$symbolizedSvg`. Please, choose the name according to the content of the variable. :-)